index.html

```diff
@@ -7,6 +7,8 @@
 	<link rel="preconnect" href="https://fonts.gstatic.com"/>
 	<link rel="preload" href="https://fonts.googleapis.com/css2?family=Roboto&family=Roboto+Slab:wght@300&display=swap" as="style"/>
 	<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&family=Roboto+Slab:wght@300&display=swap" media="print" onload="this.media='all'"/>
+	<!-- 	Please consider adding a favicon to make it look nicer -  https://i.ibb.co/wY41kQP/favicon.png -->
+    	<link rel="icon" type="image/svg+xml" href="logo.svg">
```

Comment:
The can you fix the indentation and remove the comment? Code comments are not the right place for requests

Reply:
done